core/src/main/java/org/infinispan/marshall/core/BytesObjectInput.java

```diff
@@ -1,5 +1,6 @@
 package org.infinispan.marshall.core;
```

Comment:
Probably a copy/paste error from `BytesObjectOutput` which is really expandable :)